pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -4218,3 +4220,130 @@ func (r *ClickHouseReader) CheckClickHouse(ctx context.Context) error {
 
 	return nil
 }
+
+func (r *ClickHouseReader) GetTraceAggregateAttributes(ctx context.Context, req *v3.AggregateAttributeRequest) (*v3.AggregateAttributeResponse, error) {
+	var query string
+	var err error
+	var rows driver.Rows
+	var response v3.AggregateAttributeResponse
+	// Future TODO: return filtered attributes based on aggregate operator
```

Comment:
We must not suggest matching string keys for operators such as `Max`, `Min`, `Avg`, `PXX` etc.

Reply:
Done